### DISCUSSION TITLE
jekyll-toc together with jekyll-anchor-heading

### DISCUSSION THREAD

**Discussion by xplosionmind** [allejo/jekyll-toc#37] [Q&A] [answered]
Hello, I couldn’t find any documentation to make this Liquid script work together with [jekyll-anchor-headings](https://github.com/allejo/jekyll-anchor-headings).

I’m relatively new to Jekyll and Liquid so I prefer not to mess my website up by attempting to implement both the features. What do you suggest me to do?

Should I include one include into another? Does it make sense? How can I do it?

Thanks a lot!

**Comment by allejo** [comment, marked as answer]
Don't chain the output together, you'll get weird results. Use them separately and feed the `content` variable to each snippet individually.

```liquid
<!-- Place your TOC somewhere -->
{% include toc.html html=content %}

<!-- Render your content like normally -->
{% include anchor_headings.html html=content %}
```
